python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -50,13 +60,30 @@ def config_type(cls):
                 ),
             ),
             "container_name": Field(
-                dagster.String,
+                dagster.StringSource,
                 is_required=False,
                 default_value="run",
                 description=(
                     "The container name to use when launching new tasks. Defaults to 'run'."
                 ),
             ),
+            "secrets": Field(
+                dagster.Array(dagster.String),
```

Comment:
technically this could be an array of StringSource for consistency (in practice, using env vars to determine your secrets is probably unlikely)